parsec/core/types/local_device.py

```diff
@@ -91,17 +90,21 @@ def slughash(self) -> str:
         return sha256(self.slug.encode()).hexdigest()
 
     @staticmethod
-    def load_slug(slug: str) -> Tuple[OrganizationID, DeviceID]:
+    def load_slug(slug: str) -> Tuple[str, OrganizationID, DeviceID]:
         """
         Raises: ValueError
         """
-        _, raw_org_id, raw_device_id = slug.split("#")
-        return OrganizationID(raw_org_id), DeviceID(raw_device_id)
+        rvk_hash, raw_org_id, raw_device_id = slug.split("#")
+        return rvk_hash, OrganizationID(raw_org_id), DeviceID(raw_device_id)
 
     @property
     def root_verify_key(self):
         return self.organization_addr.root_verify_key
 
+    @property
+    def root_verify_key_hash(self):
+        return sha256(self.root_verify_key.encode()).hexdigest()[:10]
```

Comment:
why not the whole hash ? (and why not return bytes ? ^^)

Reply:
> Wouldn't it be better to have two separate DeviceFileSchema classes: DeviceFileSchema and LegacyDeviceFileSchema (or DeviceFileSchemaV1 and DeviceFileSchemaV2)

I don't really have a strong opinion about it, but there's already a compatibility layer in `DeviceFileSchema`:
```
    # Since human_handle/device_label has been introduced, device_id is
    # redacted (i.e. user_id and device_name are 2 random uuids), hence
    # those fields have been added to the device file so the login page in
    # the GUI can use them to provide useful information.
    human_handle = HumanHandleField(allow_none=True, missing=None)
    device_label = fields.String(allow_none=True, missing=None)
```

Does that mean we'd need `V1`, `V2` and `V3`?  Maybe it's a bit overkill considering the amount of compatibility logic?